erts/doc/src/erlang.xml

```diff
@@ -2099,6 +2099,13 @@ true</pre>
             formatted using scientific notation with <c>Decimals</c>
             digits of precision.</p>
           </item>
+          <item><p>If option <c>short</c> is specified, the float is formatted
+            using whichever notation give the shortest amount of characters
+            that still allow a round trip conversion without any precision loss.
+            In case the float is an integer superior to 2**53, then it will be
```

Comment:
I think larger is a better term then superior here.

Is this not true for floats smaller than -2**53 as well?

Suggestion:

In case the float is larger than 2**53 or smaller than -2**53, it will be formatted using scientific notation to avoid confusing results wen doing arithmetic operations.

Reply:
Better write it as `2<sup>53</sup>` (2<sup>53</sup> in html)

Reply:
What do you think like that ?

Reply:
@DianaOlympos did you check that documentation comilation can handle the <sup> tag?

Reply:
Nope, fixed that